homeassistant/components/azure_data_explorer/__init__.py

```diff
@@ -221,12 +220,12 @@ def _parse_event(
     ) -> tuple[str | None, int]:
         """Parse event by checking if it needs to be sent, and format it."""
 
-        if not state:
+        if state is None:
             self._shutdown = True
             return None, dropped
         if state.state in FILTER_STATES or not self._entities_filter(state.entity_id):
             return None, dropped
-        if (utcnow() - time_fired).seconds > self._max_delay + self._send_interval:
+        if (utcnow() - time_fired).seconds > DEFAULT_MAX_DELAY + self._send_interval:
```

Comment:
if you just use the parameter (which is fine), then you don't need the self._max_delay in the init!